### DISCUSSION TITLE
Add imputed sequences for exportAIRR

### DISCUSSION THREAD

**Discussion by racng** [milaboratory/mixcr#1302] [Ideas]
Is there a way to export imputed sequences in AIRR format? 
Imputation for full transcript sequences is extremely helpful for things like cloning. At the same time, AIRR format is a file format that can be parsed easily by analyses software, like scirpy. It would be great if we can have both of these features in one output!

Thank you!

**Comment by racng** [comment]
Actually, I realized that I could run `extend` before `exportAIRR` and that should mostly solve the problem. The only drawback is that the imputed values are also in uppercase. 

```
mixcr extend --v-anchor L1Begin --j-anchor FR4End sample.clns sample.extended.clns
mixcr exportAirr sample.extended.clns sample.airr.extended.tsv
```

How do I add extend step to a preset file? Would the algorithm know to exportAirr/exportClones from the extended .clns file?

**Comment by mizraelson** [comment]
Hi, do you mean to add extra column to AIRR output, similar to the imputed sequence column you get with `exportClones`? We can do that.

**Reply by racng** [reply]
Instead of extra columns, can it be stored in the corresponding field without the "nImputed" prefix? For example, have the imputed sequences stored under columns: cdr1, cdr2, cdr3, fwr1, etc.? Also does the imputation also fill out the position values for start and end of these sequences as well?

**Reply by mizraelson** [reply]
That is possible, we will add this feature to the develop version in a few days.
You mean if start and end will represent the imputed sequence?

**Reply by racng** [reply]
Yes. Can we have the option to have the "_start" and "_end" fields represent the imputed sequences?
